raiden/api/python.py

```diff
@@ -476,6 +503,10 @@ def channel_open(
 
         assert channel_state, f"channel {channel_state} is gone"
 
+        self.raiden.set_channel_reveal_timeout(
```

Comment:
This works for now, but it is not atomic and will fail under restarts. The fix for this needs https://github.com/raiden-network/raiden/issues/1621